{"uuid":"10683579-d868-4fec-a2b0-2f0f63a7c267","historyId":"bcb116a57b78268555a00de608a90b72","fullName":"testCase.TC02OrderTest.placeOrderAsaGuestUser","labels":[{"name":"package","value":"testCase.TC02OrderTest"},{"name":"testClass","value":"testCase.TC02OrderTest"},{"name":"testMethod","value":"placeOrderAsaGuestUser"},{"name":"parentSuite","value":"All Test Suite"},{"name":"suite","value":"C:/Users/User/IdeaProjects/untitled1"},{"name":"subSuite","value":"testCase.TC02OrderTest"},{"name":"host","value":"DESKTOP-K9J5CR0"},{"name":"thread","value":"19988@DESKTOP-K9J5CR0.main(1)"},{"name":"framework","value":"testng"},{"name":"language","value":"java"}],"links":[],"name":"placeOrderAsaGuestUser","status":"failed","statusDetails":{"known":false,"muted":false,"flaky":false,"message":"expected [Your order has been successfully processed!\nOrder Number: 6412] but found [Your order has been successfully processed!\nOrder Number: 6418]","trace":"java.lang.AssertionError: expected [Your order has been successfully processed!\nOrder Number: 6412] but found [Your order has been successfully processed!\nOrder Number: 6418]\r\n\tat org.testng.Assert.fail(Assert.java:110)\r\n\tat org.testng.Assert.failNotEquals(Assert.java:1413)\r\n\tat org.testng.Assert.assertEqualsImpl(Assert.java:149)\r\n\tat org.testng.Assert.assertEquals(Assert.java:131)\r\n\tat org.testng.Assert.assertEquals(Assert.java:655)\r\n\tat org.testng.Assert.assertEquals(Assert.java:665)\r\n\tat testCase.TC02OrderTest.placeOrderAsaGuestUser(TC02OrderTest.java:57)\r\n\tat java.base/jdk.internal.reflect.DirectMethodHandleAccessor.invoke(DirectMethodHandleAccessor.java:103)\r\n\tat java.base/java.lang.reflect.Method.invoke(Method.java:580)\r\n\tat org.testng.internal.invokers.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:135)\r\n\tat org.testng.internal.invokers.TestInvoker.invokeMethod(TestInvoker.java:673)\r\n\tat org.testng.internal.invokers.TestInvoker.invokeTestMethod(TestInvoker.java:220)\r\n\tat org.testng.internal.invokers.MethodRunner.runInSequence(MethodRunner.java:50)\r\n\tat org.testng.internal.invokers.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:945)\r\n\tat org.testng.internal.invokers.TestInvoker.invokeTestMethods(TestInvoker.java:193)\r\n\tat org.testng.internal.invokers.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:146)\r\n\tat org.testng.internal.invokers.TestMethodWorker.run(TestMethodWorker.java:128)\r\n\tat java.base/java.util.ArrayList.forEach(ArrayList.java:1596)\r\n\tat org.testng.TestRunner.privateRun(TestRunner.java:808)\r\n\tat org.testng.TestRunner.run(TestRunner.java:603)\r\n\tat org.testng.SuiteRunner.runTest(SuiteRunner.java:429)\r\n\tat org.testng.SuiteRunner.runSequentially(SuiteRunner.java:423)\r\n\tat org.testng.SuiteRunner.privateRun(SuiteRunner.java:383)\r\n\tat org.testng.SuiteRunner.run(SuiteRunner.java:326)\r\n\tat org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)\r\n\tat org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:95)\r\n\tat org.testng.TestNG.runSuitesSequentially(TestNG.java:1249)\r\n\tat org.testng.TestNG.runSuitesLocally(TestNG.java:1169)\r\n\tat org.testng.TestNG.runSuites(TestNG.java:1092)\r\n\tat org.testng.TestNG.run(TestNG.java:1060)\r\n\tat com.intellij.rt.testng.IDEARemoteTestNG.run(IDEARemoteTestNG.java:65)\r\n\tat com.intellij.rt.testng.RemoteTestNGStarter.main(RemoteTestNGStarter.java:105)\r\n"},"stage":"finished","steps":[],"attachments":[{"name":"billing information","source":"fb16ff7e-5f48-4596-8286-19956475fab5-attachment"},{"name":"Shipping method select","source":"19013d6a-b005-45e2-b189-c67c4b1e2b40-attachment"},{"name":"Payment method select","source":"4e6ed5d0-c27c-40d3-ac2d-07d3d4b7e962-attachment"},{"name":"Order Conformation message","source":"c6cfd7ce-e02a-4279-8208-18725efba14f-attachment"}],"parameters":[],"start":1710613901234,"stop":1710614010865}